lib/rubocop/ast/node/str_node.rb

```diff
@@ -8,13 +8,56 @@ module AST
     class StrNode < Node
       include BasicLiteralNode
 
+      PERCENT_LITERAL_TYPES = {
+        :% => /^%(?![qQ])/,
+        :q => /^%q/,
+        :Q => /^%Q/
```

Comment:
It seems that matching the beginning of lines in a multiline string is not expected.

```suggestion
        :% => /\A%(?![qQ])/,
        :q => /\A%q/,
        :Q => /\A%Q/
```